app/models/position_opening.rb

```diff
@@ -252,12 +252,15 @@ def import(position_openings)
           end
         end
 
+        ts = opening[:timestamp].blank? ? DateTime.current : opening[:timestamp]
+
         client.index(
           index: INDEX_NAME,
           type: 'position_opening',
           id: "#{opening[:source]}:#{opening[:external_id]}",
           body: data.merge!({
-            timestamp: opening[:_timestamp].blank? ? DateTime.current : opening[:_timestamp],
+            timestamp: ts,
```

Comment:
I even think this code could be moved up to https://github.com/GSA/jobs_api/pull/17/commits/f522c3aee176a67e93a3491a42f1a77bb7650cb6#diff-681110cebeffad0ee72c994adfb4c087R244 as part of a case statement.